src/main/java/org/fhirvp/model/mapper/ImmunizationRecommendationMapped.java

```diff
@@ -0,0 +1,126 @@
+package org.fhirvp.model.mapper;
+
+import com.ibm.fhir.model.resource.ImmunizationRecommendation;
+import com.ibm.fhir.model.type.Reference;
+import com.ibm.fhir.path.FHIRPathNode;
+import io.vavr.control.Try;
+import org.fhirvp.Constants;
+import org.fhirvp.model.ForecastStatus;
+
+import java.util.Calendar;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class ImmunizationRecommendationMapped extends FHIRResourceMapped<ImmunizationRecommendation> {
+
+    public ImmunizationRecommendationMapped(ImmunizationRecommendation immunizationRecommendation) {
+        super(immunizationRecommendation);
+    }
+
+    public String getId() {
+        return _raw.getId();
+    }
+
+    public String getPatientId() {
+        String[] referenceParts = _raw.getPatient().getReference().getValue().split("/");
+        return referenceParts[referenceParts.length - 1];
+    }
+
+    public Calendar getCreatedDate() {
```

Comment:
Why are we using `Calendar` here and not `LocalDate`? https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/LocalDate.html

Reply:
Addressed at c5ce4bf